src/js/jsx/sections/transform/Position.jsx

```diff
@@ -53,7 +53,12 @@ define(function (require, exports, module) {
         },
 
         shouldComponentUpdate: function (nextProps, nextState) {
-            return this.state !== nextState;
+            return !this.state ||
```

Comment:
Do you think `getInitialState` will handle this better? Then you don't need the state checker here. Also, if the state is never set (just guessing), the component will always re-rendered upon receiving new props.

BTW, I see other state like `scrubX` `scrubY` and they are not check in the SCU, is it intentional?